### PR TITLE
devops: download new MacOS 12 WebKit builds

### PR DESCRIPTION
**Pull request by aslushnikov**
Drive-by: simplify management of executable paths since they are
the same for each platform.

References #11031

### CODE REVIEW

File: packages/playwright-core/src/utils/registry.ts [hunk 1]
```diff
@@ -114,6 +90,8 @@ const DOWNLOAD_PATHS = {
     'mac10.15': 'builds/firefox/%s/firefox-mac-11.zip',
     'mac11': 'builds/firefox/%s/firefox-mac-11.zip',
     'mac11-arm64': 'builds/firefox/%s/firefox-mac-11-arm64.zip',
+    'mac12': 'builds/firefox/%s/firefox-mac-11.zip',
```
Comment (dgozman) [inline packages/playwright-core/src/utils/registry.ts:93]: Just to double check - we reuse 11 build for 12?

File: packages/playwright-core/src/utils/registry.ts [hunk 2]
```diff
@@ -30,52 +30,24 @@ const BIN_PATH = path.join(__dirname, '..', '..', 'bin');
...
-    'mac11-arm64': ['pw_run.sh'],
-    'win64': ['Playwright.exe'],
+    'linux': ['pw_run.sh'],
+    'mac': ['pw_run.sh'],
+    'win': ['Playwright.exe'],
   },
   'ffmpeg': {
-    'ubuntu18.04': ['ffmpeg-linux'],
-    'ubuntu20.04': ['ffmpeg-linux'],
-    'ubuntu18.04-arm64': ['ffmpeg-linux-arm64'],
```
Comment (dgozman) [inline packages/playwright-core/src/utils/registry.ts:71]: Seems like we lost this one?

File: packages/playwright-core/src/utils/registry.ts [hunk 3]
```diff
@@ -126,6 +104,8 @@ const DOWNLOAD_PATHS = {
     'mac10.15': 'builds/firefox-beta/%s/firefox-beta-mac-11.zip',
     'mac11': 'builds/firefox-beta/%s/firefox-beta-mac-11.zip',
     'mac11-arm64': 'builds/firefox-beta/%s/firefox-beta-mac-11-arm64.zip',
+    'mac12': 'builds/firefox-beta/%s/firefox-beta-mac-11.zip',
```
Comment (yury-s) [inline packages/playwright-core/src/utils/registry.ts:107]: Should be -mac12.zip right?

Comment (aslushnikov) [reply packages/playwright-core/src/utils/registry.ts:107]: for everything but webkit we re-use macos 11 builds

File: packages/playwright-core/src/utils/registry.ts (hunk 2 above)
Comment (aslushnikov) [reply packages/playwright-core/src/utils/registry.ts:71]: :'( good catch. Let me change this to `ffmpeg-linux` for consistency.

File: packages/playwright-core/src/utils/registry.ts (hunk 1 above)
Comment (aslushnikov) [reply packages/playwright-core/src/utils/registry.ts:93]: yes we do

**Pull request merged**